### ISSUE TITLE
Lunar crashes on launch

### ISSUE THREAD

**Issue by Quinten0508** [Nilsen84/lunar-client-qt#60]
Whenever I launch Lunar using this client it immediately seems to crash, with the following error message. This same behaviour shows up on all three launch buttons. Running lunar-client-qt via the terminal and then pressing the same launch button does not give any information/logs unfortunately, I have not modified any settings or added any agents yet, and I'm using the default JRE/JDK. Issuing `java --version`gives 
```
openjdk 17.0.2 2022-01-18
OpenJDK Runtime Environment Temurin-17.0.2+8 (build 17.0.2+8)
OpenJDK 64-Bit Server VM Temurin-17.0.2+8 (build 17.0.2+8, mixed mode, sharing)
```
But I'm unsure if lunar-client-qt uses the system default java or if it uses Lunar's java - using Java v17 with MC v1.7 would not work afaik.

![image](https://user-images.githubusercontent.com/55107945/161564137-cf3aef5e-f7b8-4e35-a056-90c07bd9d483.png)

**Comment by Nilsen84** [comment]
The 1.7 crash should be fixed in LCQT 1.2.6, lmk if it works for you.

**Comment by Quinten0508** [comment]
Yup, it works now! Thanks!
